cla_public/static-src/javascripts/modules/form-errors.js

```diff
@@ -262,11 +262,19 @@
       _.each(errorFields, addErrors);
 
       if(this.$form.data('error-banner') !== false) {
-        this.$form.closest('main').prepend(this.mainFormError({ errors: this.createErrorSummary(unattachedErrors)}));
-        if (GOVUK.getCookie("locale") == "cy_GB") {
-          $("title").prepend("Gwall: ");
+        if ($("#submit-button").length && $(".s-hidden input").val()) {
+          // In case a hidden field is throwing an error (and there is a submit button),
+          $(".s-hidden input").val("");
+          // we wipe the value,
+          $("#submit-button").click();
```

Comment:
Looks good. You think the comments are necessary?

Reply:
What do you think?  I think it helps because it isn't immediately obvious why this code is here.  I suppose I could make it more succinct.  Perhaps `Remove hidden field values - LGA-1162`